pkg/bgpv1/agent/controller.go

```diff
@@ -198,7 +256,7 @@ func NewController(ctx context.Context, rtMgr BGPRouterManager, opts ...Controll
 //
 // A cancel of the provided ctx will kill the control loop along with the running
 // informers.
-func (c *Controller) Run(ctx context.Context) {
+func (c *Controller) Run(ctx context.Context, stop chan struct{}) {
```

Comment:
The new stop channel argument seems unused? Is this implementing an interface?

Reply:
Believe its passed @ https://github.com/cilium/cilium/blob/a167bc43c8d395ba7dcca7f8e0cdee4dd4776523/pkg/bgpv1/agent/controller.go#L246

Look okay?

Reply:
Cool. Marking resolved